### PR TITLE
DV Cleanup part 20

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes some travis warnings that were recently introduced.

Also fixed cleanable decals with custom set icons reverting during Initialize()

### CODE REVIEW

**Comment by praisenarsie** [comment]
99 dirty vars on the map, 99 dirty vars.
Take one down and pass it around, 98 dirty vars on the map.

**Comment by ShizCalev** [comment]
@Cyberboss Think MapDiff & IconDiff checks broke, can't get them to trigger again.

**Comment by Cyberboss** [comment]
I/O queue got choked out by the number of simultaneous renders, need to add a damn hard limit

**Comment by Cyberboss** [comment]
I fail to see how though, the maps aren't on that drive, think it's a bug, gonna try to repro

**Comment by ShizCalev** [comment]
`[19:10:25]: After rendering for  complete!`

Saw this last time too.

**Comment by Cyberboss** [comment]
Figured it out, disabled caching on the CIFS share to the VM so the host could take care of it sanely instead of quintuple queuing everything

**Comment by Cyberboss** [comment]
Also, we could potentially unit test maps for dirty vars now that i think about it, but again, like most things involving the map loader it involves separating the parser from the loader which i keep putting off cause I'm lazy

File: _maps/RandomRuins/SpaceRuins/oldteleporter.dmm [hunk 1]
```diff
@@ -63,7 +63,6 @@
 /area/ruin/space/oldteleporter)
 "n" = (
 /obj/machinery/power/apc{
-	dir = 0;
```
Comment (SpaceManiac) [inline _maps/RandomRuins/SpaceRuins/oldteleporter.dmm]: I haven't tested but there's a chance these APC and intercom changes will mess with pixel offsets. Notably APCs will be pixel offset in their specified dir and changing that from `0` to `SOUTH` will change where the APCs are positioned. Fine if all these APCs are south anyways.

Comment (ShizCalev) [reply _maps/RandomRuins/SpaceRuins/oldteleporter.dmm]: From what I can tell, they're all fine.

**Review by SpaceManiac — APPROVED** [review]
If it looks good to you then it looks good to me.

**Pull request merged**